src/prevector.h

```diff
@@ -248,32 +264,24 @@ class prevector {
 
     prevector() : _size(0), _union{{}} {}
 
-    explicit prevector(size_type n) : _size(0) {
-        resize(n);
+    explicit prevector(size_type n) : _size(n), _union{init_union()} {
```

Comment:
This is wrong. The `_size` field does not directly encode the size (see the comments above the class).

Reply:
init_union adds the `N + 1` when appropriate.

Reply:
Yuck.

Reply:
Fair enough, moved it out of the initializer.